bokeh/properties.py

```diff
@@ -332,7 +389,14 @@ def __init__(self, delegate, help="", use_prefix=True):
 """
 
 class MetaHasProps(type):
-    def __new__(cls, class_name, bases, class_dict):
+    def __new__(meta_cls, class_name, bases, class_dict):
+        # HACK for now - we get called twice for some weird reason.
+        # __new__ used to be idempotent before PropertyDescriptor
+        # had to be transformed into Property, so it was OK, but now
+        # it isn't ... I think. Anyway, on the second call just do this?
+        # we need to figure this out for real and clean it up.
+        if '__properties__' in class_dict:
+            return type.__new__(meta_cls, class_name, bases, class_dict)
```

Comment:
I have no idea what's up with this. One solution is to use only `__init__` and not `__new__`, possibly.
In any case it's not a new problem, the above hack is faster than the old code which just did everything twice (I think, at least).